docs/streams/stream-examples/realtime-updates.md

```diff
@@ -1,33 +1,40 @@
 ---
 sidebar_position: 140
-title: Realtime DB Updates Example
+title: Real-Time DB Updates
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import Prerequisites from '../../_partials/_prerequisites-sdk-api-key.md';
+import Steps from '../../_partials/_get-started-steps.md';
+import ConnectToGDN from '../../_partials/_connect-to-gdn-code-block.md';
 
-This tutorial is about using Macrometa GDN as a realtime database with local latencies across the globe.
+This tutorial demonstrates how to use Macrometa GDN as a real-time database with local latencies across the globe.
 
-## Prerequisites
+<Prerequisites />
 
-- A Macrometa account with sufficient permissions to create streams.
-- An API key. For more information, refer to [Create API Keys](../../account-management/api-keys/create-api-keys).
-- Appropriate SDK installed. For more information, refer to [Install SDKs](../../sdks/install-sdks.md).
+## Code
 
-## Realtime Updates Code
+This page guides you through creating a collection, subscribing to the collection, and automatically adding and deleting data to the collection.
+
+<Steps />
+
+If you want to skip the explanation and just run the code, then go directly to the [Full Demo File](#full-demo-file).
+
+### Step 1. Connect to GDN
+
+To update a database with Macrometa Global Data Network (GDN), you must first establish a connection to a local region.
+
+<ConnectToGDN />
+
+### Step 2. Set Variables
+
+Set the variables required to run the code.
 
 <Tabs groupId="operating-systems">
 <TabItem value="js" label="Javascript">
 
 ```js
```

Comment:
Please work with @koshy-thomas to get some log lines into the JavaScript steps. At least for the first three steps, there is no user response, which is unsatisfying and worrisome to a user going through this step by step.

Be sure to make the same changes to the full demo if appropriate.